modules/Attendance/attendance_take_byCourseClass.php

```diff
@@ -106,7 +106,7 @@
 
                 // Check class
                 try {
-                    $data = array("gibbonCourseClassID" => $gibbonCourseClassID, "gibbonSchoolYearID" => $_SESSION[$guid]["gibbonSchoolYearID"]);
+                    $data = array("gibbonCourseClassID" => $gibbonCourseClassID, "gibbonSchoolYearID" => $session->get("gibbonSchoolYearID"));
```

Comment:
For consistency, we do prefer single quotes (string literal) over double quotes here. It's not a syntax issue, but it's nice to have this consistent in the codebase, so see if you can change these going forward.
```suggestion
                    $data = array("gibbonCourseClassID" => $gibbonCourseClassID, "gibbonSchoolYearID" => $session->get('gibbonSchoolYearID'));
```